package.json

```diff
@@ -104,7 +104,7 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-regexp": "^1.15.0",
     "husky": "^8.0.3",
-    "image-size": "^1.0.2",
```

Comment:
This will only upgrade our monorepo, not third-party Docusaurus sites

Our mdx-loader package should also upgrade to v2 otherwise this creates a dependency duplicate that we won't even use

Reply:
I've seen the fix on 1.2.1 but when I check out `image-size` node_modules folder there is `1.2.0` used:

<img width="758" alt="image" src="https://github.com/user-attachments/assets/8ddcec92-91de-4fc4-b9cd-fb596768decb" />

I'm pretty sure you are totally right by what you are telling me, it's just tools that seek for vulnerabilities like snyk will list you because of this exploit and this give people a bad meaning if they do not understand fully what happens.

BTW, I do not have a CLA and I don't know if my organization wants to do this again, this has been a pain doing it for AMP decades ago, but I still updated `mdx-loader` and pushed it as you can see